extensions/commands/art/cmd_build_info.py

```diff
@@ -142,6 +143,12 @@ def get_artifacts(self, node, artifact_type, is_dependency=False):
 
         def _get_local_artifacts():
             local_artifacts = []
+            binary_skipped = node.get("binary") == "Skip"
+            no_pkg_folder = node.get("package_folder") is None
+            if binary_skipped and no_pkg_folder:
```

Comment:
I am not sure skipped binaries should error by default. It should be possible to construct the buildInfo of a dependency graph, excluding the skipped binaries (typically tool-requires)